src/constraint.rs

```diff
@@ -0,0 +1,28 @@
+use codec::{Decode, Encode};
+use rstd::collections::btree_set::BTreeSet;
+
+/// Pointer to a specific property of entities of a specfic class.
```

Comment:
There is nothing about entity in this structure. You only refer to class and property index.
```suggestion
/// Pointer to a specific property of a specific class.
```

Reply:
I mean that property is declared in class, whereas property value is a part of an entity.
So, in this case, it's just a property of a class.